app/scripts/controllers/LoginCtrl.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name frontendApp.controller:LoginCtrl
+ * @description
+ * # loginCtrl
+ * Controller of the frontendApp
+ */
+angular
+  .module('frontendApp')
+  .controller('LoginCtrl', ['$scope', 'UserModel', function ($scope, $userModel) {
+
+    $scope.user = null;
+
+    $scope.doLogin = (function(user) {
```

Comment:
doLogin creates a user?
Maybe would be better as well to name consistently the front end with the backend. Use SignUp,SignIn and SignOut